tempesta_fw/http_parser.c

```diff
@@ -881,9 +900,9 @@ enum {
 	Req_UriSchHttp,
 	Req_UriSchHttpColon,
 	Req_UriSchHttpColonSlash,
-	Req_UriHostStart,
-	Req_UriHost,
-	Req_UriHostEnd,
+	Req_UriAuthorityStart,
+	Req_UriAuthority,
+	Req_UriAuthorityEnd,
```

Comment:
It'd better to name the states like Req_UriAuthHost* since they process host part of URI as well as authority